src/components/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import css from "./SignUp.module.css";
+import {customHistory} from "../../App";
+import {ErrorMessage} from "../Errors/ErrorMessage/ErrorMessage";
+import {ErrorValidation} from "../Errors/ErrorValidation/ErrorValidation";
+
+export class SignUp extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            form: {
+                name: '',
+                email: '',
+                password: '',
+                repeatPassword: ''
+            },
+            responseStatus: '',
+            errorValidation: {
+                name: '',
+                email: '',
+                password: '',
+            },
+            errorMessage: ''
+        }
+    }
+
+    isEmpty = (str) => {
```

Comment:
Might be simpliefied to `isEmpty = str => !!str.trim()

And the name of this function misleading a bit) souds like:
- Hi `isEmpty`, is the string `""` empty??
- *isEmpty evaluates `false`* no, this string empty!
- But.. it... is.. empty...
- No!!